styles/Testimonial.module.css

```diff
@@ -78,18 +89,19 @@
   margin: 5%;
   object-fit: cover;
 }
+
 .testimonial__content > h5 {
   color: #d7e1ec;
   line-height: 1.7;
   font-weight: 500;
   letter-spacing: 0;
 }
 
-.testimonial__content > p {
+.testimonial__content  p {
```

Comment:
is this part of your change? you are altering the current code here? please update me with the impact